backend/drm/drm.c

```diff
@@ -635,18 +645,18 @@ static bool drm_connector_export_dmabuf(struct wlr_output *output,
 
 	struct wlr_drm_plane *plane = crtc->primary;
 
-	if (plane->current_fb.type == WLR_DRM_FB_TYPE_NONE) {
+	if (!plane->current_fb.bo) {
 		return false;
 	}
 
 	return export_drm_bo(plane->current_fb.bo, attribs);
```

Comment:
The frame now comes y-inverted from screencopy-dmabuf. Works fine with shm screencopy, and even yields 2% less CPU load while doing so on my system than it did before. 

One way to fix the y-inversion would be to set `attribs->flags |= WLR_DMABUF_ATTRIBUTES_FLAGS_Y_INVERT` here, but software that uses export-dmabuf and doesn't check for y-inversion might suffer.

Reply:
Hmm, interesting, because with the wlroots example I get a video with the correct orientation. What client are you testing this with?

Reply:
I tested with wayvnc, which does take y-inversion of buffers into account. Did you test with the screencopy-dmabuf sample client or the export-dmabuf sample client?

Reply:
Oh, my bad, I tested with dmabuf-capture, not with screencopy-dmabuf. I'll investigate.

Reply:
`blit_dmabuf` assumed that the renderer flips the texture, but that isn't necessarily the case anymore. Fixed.